.vscode/launch.json

```diff
@@ -36,21 +36,6 @@
             "stopOnEntry": true
         },
         {
-            "name": "Run Tests",
-            "type": "node",
-            "request": "launch",
-            "program": "${workspaceRoot}/node_modules/mocha/bin/_mocha",
-            "args": [
-                "./out/tests",
-                "--timeout",
-                "999999",
-                "--colors"
-            ],
-            "sourceMaps": true,
-            "outDir": "${workspaceRoot}/out/",
-            "preLaunchTask": "build"
```

Comment:
Don't we still need the preLaunchTask to make sure that tests are compiled before running them?